app/src/main/java/com/besome/sketch/beans/ProjectLibraryBean.java

```diff
@@ -77,6 +83,8 @@ public ProjectLibraryBean(Parcel parcel) {
         parcel.readTypedList(adUnits, AdUnitBean.getCreator());
         testDevices = new ArrayList<>();
         parcel.readTypedList(testDevices, AdTestDeviceBean.getCreator());
+        material3Configurations = new HashMap<>();
+        parcel.readMap(material3Configurations, HashMap.class.getClassLoader());
```

Comment:
The HashMap would use the same classloader as every other class, you should use `ProjectLibraryBean.class.getClassLoader()` instead